App.js

```diff
@@ -1,18 +1,58 @@
 import React from 'react';
-import { Text, View } from 'react-native';
-import { createBottomTabNavigator, createAppContainer } from 'react-navigation';
-import {HomeScreen} from './Home';
-import {MediaScreen} from './Media';
-import {GroupsScreen} from './Groups';
-import {EngageScreen} from './Engage';
-import {GivingScreen} from './Giving';
+import { Platform, StatusBar, StyleSheet, View } from 'react-native';
+import { AppLoading, Asset, Font, Icon } from 'expo';
+import AppNavigator from './navigation/AppNavigator';
 
-const TabNavigator = createBottomTabNavigator({
-  Home: HomeScreen,
-  Media: MediaScreen,
-  Groups: GroupsScreen,
-  Engage: EngageScreen,
-  Giving: GivingScreen
-});
+export default class App extends React.Component {
+  state = {
+    isLoadingComplete: false,
+  };
+
+  render() {
+    if (!this.state.isLoadingComplete && !this.props.skipLoadingScreen) {
+      return (
+        <AppLoading
+          startAsync={this._loadResourcesAsync}
+          onError={this._handleLoadingError}
+          onFinish={this._handleFinishLoading}
+        />
+      );
+    }
+    return (
+      <View style={styles.container}>
+        {Platform.OS === 'ios' && <StatusBar barStyle="default" />}
+        <AppNavigator />
+      </View>
+    );
+  }
+
+  _loadResourcesAsync = async () => {
+    return Promise.all([
+      Asset.loadAsync([require('./assets/images/echo_logo.png')]),
```

Comment:
async load the echo logo image, so we have have it before render

Reply:
Nice! Although we eventually need to make sure we know what happens if there's no (or very slow) internet.

Reply:
yeah, for sure

Reply:
I stuck it in the backlog--a problem for later.